src/services/item/plugins/app/appAction/schemas.ts

```diff
@@ -1,15 +1,13 @@
-import { UUID_REGEX } from '../../../../../schemas/global';
-
 export default {
   $id: 'https://graasp.org/apps/app-action/',
   definitions: {
     appAction: {
       type: 'object',
       properties: {
         id: { type: 'string' },
-        member: {
-          $ref: 'https://graasp.org/members/#/definitions/member',
-        },
+        account: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
+        /** @deprecated use account */
+        member: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
```

Comment:
I think we should mark the JSONSchema as deprecated: https://stackoverflow.com/questions/49870109/how-to-annotate-a-field-as-deprecated-in-openapi-swagger-2-0
Sop it shows on OpenAPI documentation.